### PR TITLE
Thuy customer detail (completed)

### LINKED ISSUE #72: Implment customer detail page

### PR DESCRIPTION
**Pull request by yuhtnguyen**

### CODE REVIEW

File: src/main/java/io/hardingadonis/saledock/controller/management/customer/CustomerDetailServlet.java [hunk 1]
```diff
@@ -1,16 +1,36 @@
 package io.hardingadonis.saledock.controller.management.customer;
 
+import io.hardingadonis.saledock.model.Customer;
```
Comment (hardingadonis) [inline src/main/java/io/hardingadonis/saledock/controller/management/customer/CustomerDetailServlet.java]: Use ```import *;```

File: src/main/java/io/hardingadonis/saledock/controller/management/customer/CustomerDetailServlet.java [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+import io.hardingadonis.saledock.utils.Singleton;
 import java.io.*;
 import jakarta.servlet.*;
 import jakarta.servlet.annotation.*;
 import jakarta.servlet.http.*;
+import java.util.Optional;
```
Comment (hardingadonis) [inline src/main/java/io/hardingadonis/saledock/controller/management/customer/CustomerDetailServlet.java]: Use `import *;`

File: src/main/java/io/hardingadonis/saledock/controller/management/customer/CustomerDetailServlet.java [hunk 3]
```diff
@@ -1,16 +1,36 @@
 package io.hardingadonis.saledock.controller.management.customer;
 
+import io.hardingadonis.saledock.model.Customer;
+import io.hardingadonis.saledock.utils.Singleton;
```
Comment (hardingadonis) [inline src/main/java/io/hardingadonis/saledock/controller/management/customer/CustomerDetailServlet.java]: Use `import *;`

**Review by hardingadonis — APPROVED** [review]

**Review by thson58 — APPROVED** [review]

**Review by bakaqc — APPROVED** [review]
not bad

**Pull request merged**